mbtb_app/web_portal/mbtb_portal/views/pages/admin_view_current_users.ejs

```diff
@@ -58,15 +59,18 @@
                         <td><%= data[i].postal_code %></td>
                         <td><%= data[i].comments %></td>
                         <td><%= data[i].active_since %></td>
+                        <td><%= data[i].revert_reason %></td>
                     </tr>
                 <% } %>
             </table>
         </form>
 
         <br>
         <br>
+
         <div class="form-group button_holder">
             <div class="col" id="cancel_btn">
+                <a href="/suspend_user_form" class="btn btn-primary">Go to suspend form</a>
```

Comment:
No need to add this line, we will call sails controller in js file, to send data. remove it.